postgres/osmap.yaml

```diff
@@ -85,15 +90,30 @@ RedHat:
 
 {% else %}
 
+  {% set data_dir = '/var/lib/pgsql/data' %}
   pkg: postgresql-server
   pkg_client: postgresql
 
 {% endif %}
 
+  service: postgresql-{{ repo.version }}
+  pkgs_extra: [ postgresql-contrib, postgresql-plpython, postgresql-jdbc, postgresql-docs]
+  conf_dir: {{ data_dir }}
+  prepare_cluster:
+    command: initdb --pgdata='{{ data_dir }}'
+    test: test -f '{{ data_dir }}/PG_VERSION'
+    user: postgres
+
 Suse:
   pkg: postgresql-server
   pkg_client: postgresql
   pkg_libpq_dev: postgresql
+  pkgs_extra: [ postgresql-contrib, postgresql-plpython, postgresql-jdbc, postgresql-docs]
 
+  {% set data_dir = '/var/lib/pgsql/data' %}
+  prepare_cluster:
+    command: /usr/lib/postgresql/bin/initdb --pgdata={{ data_dir }}
```

Comment:
I guess this is the only required change for Suse. Are you sure that `initdb` is not in `$PATH`?

Reply:
The command failed on SuSE so I think I found the binary in that path.